### ISSUE TITLE
If too many winks get sent off, they can't all be replaced 10cm apart

### ISSUE THREAD

**Issue by timhunt** [etwa-rsc/rules#6]
From the venerable Bobo's rules problems.docx:

A quick calculation on the back of an envelope will make it clear that this is impossible in some bizarre situations. Even in non-bizarre situations, the ten-centimetre rule can cause winks to be placed much further from or closer to the pot than they would otherwise be placed. This sometimes seems unfair, especially when the 10cm minimum is being maintained between friendly winks. Matt suggests two solutions.

1. Winks going off are replaced as near as possible to the point where they went off subject to being 22mm from the boundary, at least 10cm away from opposing winks (and opposing baselines with winks behind them) and 2mm from friendly winks (and friendly baselines with winks behind them). The problem with this suggestion is that it can create doubletons of friendly winks which go off near each other; perhaps 22mm would be better than 2mm. 

2. Winks going off are replaced as near as possible to the point where they went off subject to being at least 22mm from the boundary and at least 10cm from any other wink (or baseline with winks behind it).

**Comment by timhunt** [comment]
Although this could happen, it requires more than half the winks in the game to be sent off in a single shot, whereas it is rare for more than one or two to be sent off. So we don't really need to worry about this.

**Comment by timhunt** [comment]
The current wording of the relevant rule is:

> 13.1 The wink should be placed flat on the mat in a position as near as possible to the point at which it left the playing volume subject to: (i) its nearest edge should be 22 mm in from the boundary to the field of play; (ii) its nearest edge should not be closer than 10 cm to the edge of any other wink; (iii) its nearest edge should not be closer than 10 cm to any baseline with an unplayed wink behind it.

**Comment by fluppeteer** [comment]
The wording of this rule did cause some confusion in translation, although I don't personally think it's ambiguous. I agree that we can ignore it for practical purposes.

I think I'm missing why all the winks had to be sent off in a single shot?

**Comment by hatj2** [comment]
Yes - it can happen if lots of winks get sent off in subsequent shots without previous ones benig brought in, of course. 
I like Matt's option 1) with the 22mm amendment, if we think we ought to change this; not all situations are solved by this, though, obviously.

**Comment by fluppeteer** [comment]
I'd be happy with that. (I don't really mind even if it's a 2mm version - if you were so careless going off that it's in your opponents' interests to squop you on the baseline... although I imagine big arguments about whether the winks were really that close together when they went off in that case, so maybe 22mm would reduce ranting.)

**Comment by hatj2** [comment]
I think you'd only be putting them that close together in the case where it was difficult to position a big pile, all of which went off. Winks that had been sent off and positioned at the edge in previous turns could be movable if it helps the distribution of winks sent off subsequently, perhaps?
